nuitka/Tracing.py

```diff
@@ -81,7 +81,7 @@ def my_print(*args, **kwargs):
 
         del kwargs["style"]
 
-        print(style, *(args + ("\033[0m",)), **kwargs)
+        print(*((style + args[0],) + args[1:] + ("\033[0m",)), **kwargs)
```

Comment:
Ah, so this is to address the leading spaces that I saw, and never understood.

You don't have to force yourself into doing everything in one print statement, also, image something does print something that is not a string, say an object, then `+` would fail.

You can just do `print(style, end="")` before the real print, and then `print("\033[0m", end="") ` afterwards, that is probably easier?